### ISSUE TITLE
dns.name.EmptyLabel: A DNS label is empty.

### ISSUE THREAD

**Issue by abhiramk7** [naokishibuya/car-behavioral-cloning#29]
When I tried to run the drive.py file using the code _python drive.py model.h5_  
it shows the error as **dns.name.EmptyLabel: A DNS label is empty.**.How can I solve this?.
advanced thanks.

**Comment by SallahiNarjiss** [comment]
Hello please did you solve it ?

**Comment by AladdinDorado** [comment]
Yes but you have to enrol at Udemy and buy the course Self Driving
Car-complete course - it's awesome for a low price.
